WireAuthentication/Sources/WireAuthenticationUI/Views/DetermineAuthMethod/DetermineAuthMethodView.swift

```diff
@@ -121,10 +128,11 @@ package struct DetermineAuthMethodView: View {
         .presentationDragIndicator(.hidden)
     }
 
-    enum Destination: Hashable {
+    package enum Destination: Hashable {
 
         case login(email: String)
         case loginOrRegister(email: String)
+        case noHistory(userID: UUID, cookieData: Data)
```

Comment:
question: I thought that `noHistory` is actually a destination of the `RootView`.